store/sqlstore/post_acknowledgements_store.go

```diff
@@ -86,11 +108,21 @@ func (s *SqlPostAcknowledgementStore) Delete(ack *model.PostAcknowledgement) err
 			sq.Eq{"UserId": ack.UserId},
 		})
 
-	_, err := s.GetMasterX().ExecBuilder(query)
+	_, err = transaction.ExecBuilder(query)
 	if err != nil {
 		return err
 	}
 
+	err = updatePost(transaction, ack.PostId)
```

Comment:
nit: slightly weird to have this one called `ack` and elsewhere called `acknowledgement`